lib/cfn-nag/custom_rules/EC2NetworkAclEntryPortRangeRule.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'base'
+
+class EC2NetworkAclEntryPortRangeRule < BaseRule
+  def rule_text
+    'EC2 NetworkACL Entry Port Range must be defined for TCP/UDP protocols and not allow all ports'
+  end
+
+  def rule_type
+    Violation::FAILING_VIOLATION
```

Comment:
i'm thinking this is probably better as a warning?  in my mind, it's legit to develop VPCs that pass the traffic, and security groups bear the brunt.

Reply:
The only issue that may arise here is that technically the Port Range is required for TCP and UDP protocols.  Should that be a separate rule or not addressed at all?